aip/general/0130.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 130
+state: draft
+created: 2023-03-313
+updated: 2023-03-313
+placement:
+  category: operations
+  order: 9
+---
+
+# Methods
+
+Methods are conceptually synonymous with
+
+## Guidance
+
+### Categories of Methods
+
+The following enumerates multiple categories of methods that exist, often
+grouped up under some object (e.g. collection or resource) that the method
+operates upon.
+
+| Category Name                      | Description                                                                                                                                                                                                  | Related AIPs                                       | [IaC][]                         | CLIs        | UIs         | SDKs        |
```

Comment:
I know this formatting is *really* hard to read. I'd love some suggestions on how to author a more legible markdown table.

Reply:
I just view the rendered file.